homeassistant/components/tuya/sensor.py

```diff
@@ -321,10 +325,36 @@
         SensorEntityDescription(
             key=DPCode.GAS_SENSOR_VALUE,
             icon="mdi:gas-cylinder",
-            device_class=STATE_CLASS_MEASUREMENT,
+            device_class=DEVICE_CLASS_GAS,
+            state_class=STATE_CLASS_MEASUREMENT,
         ),
         *BATTERY_SENSORS,
     ),
+    # Smart Water Timer
+    "sfkzq": (
+        SensorEntityDescription(
+            key=DPCode.BATTERY_PERCENTAGE,
+            name="Battery",
+            native_unit_of_measurement=PERCENTAGE,
+            device_class=DEVICE_CLASS_BATTERY,
+            state_class=STATE_CLASS_MEASUREMENT,
+            entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
+        ),
+        SensorEntityDescription(
+            key=DPCode.USE_TIME_ONE,
+            name="Use time",
+            native_unit_of_measurement=TIME_SECONDS,
+            device_class=DEVICE_CLASS_TIMESTAMP,
+            state_class=STATE_CLASS_TOTAL_INCREASING,
+            entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
+        ),
+        SensorEntityDescription(
+            key=DPCode.WORK_STATE,
+            name="Work state",
+            state_class=STATE_CLASS_MEASUREMENT,
```

Comment:
Is a work state a measurement?

Reply:
state class can be None if the available state classes don't fit

Reply:
thanks